php/libraries/User.class.inc

```diff
@@ -525,6 +527,60 @@ class User extends UserPermissions
         $this->update($updateArray);
     }
 
+    /**
+     * Use the Pwned Passwords API to determine whether a user is trying
+     * to use a password that has been revealed in public data breaches.
+     *
+     * @param string $password The password to check for pwnage.
+     *
+     * @return bool Whether the password is known to be pwned.
+     */
+    function pwnedPassword(string $password): bool
+    {
+        /* The pwned passwords API searches a range of password hashes to
```

Comment:
```suggestion
        /*
         * The pwned passwords API searches a range of password hashes to
```

Reply:
Is it worth adding a PHPCS rule for this? I didn't know this was a style thing we had.